src/pages/appbuilder/AppBuilderPage.tsx

```diff
@@ -18,6 +18,94 @@ import AppBuilderTemplateSelector from "pages/templates/AppBuilderTemplateSelect
 
 const VIEWPORT_ID = "viewport_1";
 
+const WelcomePlatformMarkdown = `
+## Welcome to the ShapeDiver App Builder
+
+You can use this page to display any model from your ShapeDiver platform [library](https://help.shapediver.com/doc/model-library), 
+as well as public models. You will be redirected to the login page if you are not logged in to the platform.
+
+You can display:
+
+   * models owned by your account (including private models), 
+   * models shared with you, and
+   * public models.
+
+Note: You do **not** need to enable iframe or direct embedding for this to work.
+
+Example: 
+
+[${window.location}?slug=react-ar-cube](${window.location}?slug=react-ar-cube)
```

Comment:
@mathieu-shapediver we should use another example model, which actually has some App Builder functionality. Please suggest one (should be stored on the demo account).

Reply:
We can change this later